ui-lib/src/main/res/ui/settings/values/strings.xml

```diff
@@ -1,11 +1,14 @@
 <resources>
+    <string name="settings_title">SETTINGS</string>
```

Comment:
The same comment on the capitalization here.

Reply:
done